cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -182,18 +182,20 @@
 
   <property>
     <name>twill.java.heap.memory.ratio</name>
-    <value>0.6</value>
+    <value>0.7</value>
```

Comment:
Keep this at 0.6. It applies to application containers as well.

Reply:
this is per the jira: https://issues.cask.co/browse/CDAP-12767, and to keep configs consistent across CM/Ambari/Packages.  I guess an alternative is to also specify all ``[service].twill.java.heap.memory.ratio`` also in cdap-default.xml...

Reply:
reverting to 0.6... there was a typo in the jira.